RELEASE.md

```diff
@@ -2,22 +2,59 @@
 
 [![hackmd-github-sync-badge](https://hackmd.io/sVOAyv6LTwiQllQUctxP1w/badge)](https://hackmd.io/sVOAyv6LTwiQllQUctxP1w)
 
-## Resolve Issues
+## Versioning Scheme
+Flyte components follow [SemVer conventions](https://semver.org/). In a nutshell, that means every component has a release version that follows this format: `MAJOR.MINOR.PATCH`. This includes the aggregated milestone release described below.
+
+Flyte hasn't reached `v1.0.0` version yet. This version indicates a stable public API. After releasing `v1.0.0` any backward incompatible changes require a `MAJOR` version bump. The community and the team, however, strives to maintain backward compatibility with the current public APIs as much as possible and gracefully deprecating APIs when absolutely required.
```

Comment:
I want to be careful here. I want to say that we have not cut a 1.0, though ideally we are 1.0, we have not really released a 2.0 right?
i.e, we are stable and production ready. The absence of 1.0 does not imply we are not production ready. Can we clarify that here